opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "1.2"
+maintainer: "Sylvain Le Gall <gildor@ocamlcore.org>"
+authors: [ "Sylvain Le Gall" ]
+license: "LGPL-2.1 with OCaml linking exception"
+homepage: "http://oasis.forge.ocamlcore.org/"
+dev-repo: "git://github.com/gildor478/oasis.git"
+bug-reports: "http://oasis.forge.ocamlcore.org/"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix
+    "--%{gettext:enable}%-gettext"]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
+  # If _oasis_remove_.ml is not present, this is an old installation
+  # and one needs to perform the removal "manually".  The next version of
+  # the package will use the first method only.
+  ["/bin/sh" "-c" "if [ -e '%{etc}%/oasis/_oasis_remove_.ml' ]; then ocaml '%{etc}%/oasis/_oasis_remove_.ml' '%{etc}%/oasis'; else ocamlfind remove plugin-loader; ocamlfind remove userconf; ocamlfind remove oasis; rm '%{bin}%/oasis'; fi"]
```

Comment:
1. This will not work on Windows.
2. This is not future proof: when the next version of oasis is released (the one that properly supports `-C`), `_oasis_remove_.ml` will not be necessary anymore.